schemas/fingerprint-server-api.yaml

```diff
@@ -1771,3 +1782,14 @@ components:
       required:
         - code
         - message
+    RemoteControlResult:
+      type: object
+      additionalProperties: false
+      properties:
+        result:
+          type: boolean
+          description: >
+            `true` if the request came from a machine being remotely controlled (e.g. TeamViewer) or remotely controlling another machine, `false` otherwise.
```

Comment:
I would skip the part with "remotely controlling another machine" - although true it doesn't reflect the use cases we're targetting.